bdb/rep.c

```diff
@@ -3209,6 +3211,9 @@ static int bdb_wait_for_seqnum_from_all_int(bdb_state_type *bdb_state,
                     bdb_state->repinfo->netinfo, connlist);
         }
 
+        if (gbl_debug_all_incoherent && (rand() % 2)) {
```

Comment:
Sorry to nitpick but I already wasted the time to look at the usage of j and total_connected in this function and can't resist: seems j is unnecessary variable and can be removed, you can use total_connected instead of j. And your code can just be if (xxx) goto done_wait;

Reply:
sure no worries nitpick away ..